Objects/descrobject.c

```diff
@@ -1634,6 +1670,8 @@ property_copy(PyObject *old, PyObject *get, PyObject *set, PyObject *del)
     Py_DECREF(type);
     if (new == NULL)
         return NULL;
+
```

Comment:
I think there should be an incref on `pold->prop_name`.

Try running a debug build to whether the refcounts are being driven negative.

Reply:
You are totally right, ref count must be increased